src/Qir/Execution/CommandLineCompiler/CommandLineCompiler.csproj

```diff
@@ -0,0 +1,13 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="System.CommandLine" Version="2.0.0-beta1.20213.1" />
+    <ProjectReference Include="..\Tools\Microsoft.Quantum.Qir.Tools.csproj" />
```

Comment:
Microsoft.Quantum.Qir.Tools

Reply:
Thanks! I've updated the reference now.

Reply:
Where would this move to? I think it makes sense to have this where the qir tools are, too, and then it would be better if it was built against the source code version of that package. Similarly for all the package references to the other packages that are built on this repo; I don't think having them as package references will work well with our e2e build, and I don't see a reason why they would need to be project references. Upon packaging this, we can configure that the project references are replaced by package references. What is your thinking here, @cesarzc?

Reply:
Yes, it makes sense to have it here. I agree then that package references should be changed to project references and depending on how we package this tool, we might not even have to change them to package references at any point.

Reply:
Which project does the `Microsoft.Quantum.Qir.Runtime` package refer to?

Reply:
`Microsoft.Quantum.Qir.Runtime` package does not have a corresponding project file since it's all native binaries and headers. Since this in mostly relevant for packaging, for this PR the package reference can be removed.

Reply:
removed